### ISSUE TITLE
Wrong phishing detection for mycrypto.tools website

### ISSUE THREAD

**Issue by tunatore** [MetaMask/eth-phishing-detect#3501]
Hi, 
Metamask displays wrong phishing warning for the following website; 
https://mycrypto.tools/ https://www.mycrypto.tools/ 
This website is used to educate people about cryptocurrencies.
Thanks and regards

**Comment by tunatore** [comment]
Hi,
Any update on this, could you please prioritise this?
Regards,

**Comment by 409H** [comment]
Hi @tunatore - your domain is not explicitly blacklisted but is being caught on the fuzzy match against `mycrypto.com`. I had a quick look at your site and with the address generation tools, whilst you say you don't store secrets, the secrets are sent from your server with a `POST https://ux84sd384k.execute-api.us-east-1.amazonaws.com/btcethStage` request

> *Please be informed that we never store keys and addresses related to this Ethereum address generation on our web servers.

Can you expliclity say how the keys are generated (ie: that they are generated server side and not client side)

**Comment by tunatore** [comment]
Hi, @409H. 

Thank you for your response and investigation. The post-stage you mentioned at https://ux84sd384k.execute-api.us-east-1.amazonaws.com/btcethStage is used to call amazon web service to generate the actual address and not for storing. The post method returns the actual address from the server using AWSApiGatewayResponse.

If we check it in details, in the POST request the following parameters are passed to the server:

**1. 'method': "btc-eth-address-create",** --> indicates that this method should create a coin address
**2. 'coin': "BTC",** --> indicates that coin type should be BTC
**3.  'captcha': grecaptcha.getResponse()** --> we sent the google captcha; this is standard google captcha response, which is required to prevent the service from being called by robots.

As you can see from the above parameter lists, these are quite standard parameters. We don't pass or sent any address details on this page; moreover, we mentioned that we don't have logging on the server-side for the address generated on the servers. Our main aim with this website is to educate users with a  technical course which is still under development but it will be finalized this month. We also mentioned; "You may use the generated BTC - Bitcoin - address, however, it is not recommended and it is not for personal usage." To have the complete ownership of BTC - Bitcoin - address generation, it is advised to attend the following course. Please click here.

You can also check the following link explaining the Bitcoin address generation steps in details:
https://mycrypto.tools/btcaddress.html

Basically, with this website, we would like to educate people using technical explanations. You can see our architecture at this blog posts if you need more information about the services which is going to be provided at https://mycrypto.tools

https://mycrypto.tools/blog_how_mycrypto_tools_uses_aws_lambda_serverless_architecture.html
https://mycrypto.tools/blog_how_to_generate_bitcoin_addresses.html

Thanks again for your time and support on this issue.

Regards,
Tuna Tore

**Comment by tunatore** [comment]
Hi @409H ,
When can we expect a resolution to this wrong phishing detection.

Some of my prospective users complained about this and they think that my website could be a phishing site. mycrypto.tools website is used to promote my upcoming course about cryptocurrencies. 

I think it is good for the community to remove this false positive happening in Ethereum Phishing Detection. It should be an easy patch in fuzzy match. 

Thanks for your assistance on this issue.
